Configure

```diff
@@ -2250,12 +2257,14 @@ EOF
                     $unified_info{sources}->{$o}->{$s} = -1;
                 } elsif ($s =~ /\.rc$/) {
                     # We also recognise resource files
+                    push @{$check_exist{$s}}, $ddest;
                     my $o = $_;
                     $o =~ s/\.rc$/.res/; # Resource configuration
-                    my $o = cleanfile($buildd, $o, $blddir);
+                    $o = cleanfile($buildd, $o, $blddir);
                     $unified_info{sources}->{$ddest}->{$o} = -1;
                     $unified_info{sources}->{$o}->{$s} = -1;
                 } else {
+                    push @{$check_exist{$s}}, $ddest;
                     $unified_info{sources}->{$ddest}->{$s} = 1;
                 }
```

Comment:
Independently of the improvements done here, 
shouldn't this last `else` case become some `elsif` case followed by:
```
                } else {
                    die "unrecognised source file type for shared library: $s\n";
                }
```
as done for the `foreach` loop a little further down?

Reply:
No, because we allow other stuff as source as well.  For example, static libraries (using static libraries as a source mimics the [Cmake object libraries](https://cmake.org/cmake/help/v3.19/command/add_library.html#object-libraries)

I've left that lose ended, to leave it to the build file templates to determine exactly what they accept, for now.